src/main/java/com/google/alpollo/EntityServlet.java

```diff
@@ -17,13 +18,11 @@
 @WebServlet("/entity")
 public class EntityServlet extends HttpServlet {
   private final Gson gson = new Gson();
-  /** Parameter sent through the request which holds the song's lyrics. */
-  private static final String LYRICS = "lyrics";
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     try {
-      String lyrics = request.getParameter(LYRICS);
+      final String lyrics = gson.fromJson(request.getReader(), Lyrics.class).getLyrics();
```

Comment:
Is it not possible to keep lyrics as String directly?
Something like 
`final String lyrics = gson.fromJson(request.getReader(), String.class);`
That would remove the need for the Lyrics class which is just a wrapper around String.

Reply:
Yes, we could, but in this case we need to change not only parsing of the body, but the format of request that we send from JSON to smth else, since single line JSON usually not considered as valid (if it's not array from 1 element, of course). So I thought that it would be better just to change parsing 🤔